cms/cms-import.md

```diff
@@ -0,0 +1,272 @@
+---
+title: Configuring Imports and Exports
+---
+
+[[toc]]
+
+Handling bulk data in Skygear CMS is easy. Your users can import data with a CSV and export data to a CSV. 
+
+## Imports
+
+![Skygear CMS import](/assets/cms/cms-import.png)
+
+Your users can import data to Skygear with a CSV. To do so, you need to configure the import settings and add an import button to the right page.
+
+### Configuring imports
+
+There are a few items you can configure in the import settings, and we are using the following 2 tables to illustrate how.
+
+**Blogpost**
+|Fields| Type| 
+|---|---|
+|_id| String|
+|_created_at| DateTime|
+|_updated_at| DateTime|
+|title| String|
+|content| String|
+|cover| Asset|
+
+**Comment**
+|Fields| Type| 
+|---|---|
+|_id| String|
+|_created_at| DateTime|
+|_updated_at| DateTime|
+|comment| String|
+|blogpostId | Reference|
+
+This is what we need:
+
+1. Allow users to import a list of comments. 
+2. We will use `comment` as an identifier. When there are two identical comments, they are considered as duplicated records.
```

Comment:
should be "use `content` as an identifier"

Reply:
More accurate usage of `identifier` should be, we will update the record if we found a record with the same identifier in db otherwise we will insert a new record. `handle_duplicated_identifier` is the way to handle if we found more than 1 record from the db. `throw-error` is no update, `use-first` pick one randomly and update.

Reply:
Here I am using the 'comment' field as an identifier but it looks to me that the example is not clear enough. Hmmm let me see how to fix it.

Upsert: noted.

Reply:
@carmenlau can we import _id?

Reply:
fixed.